ReleaseNotes2_3.md

```diff
@@ -10,23 +10,26 @@ This document describes the changes and updates that have been made in version 2
  - The `EN_getaveragepatternvalue` function will now accept a pattern index of zero which represents the constant pattern assigned to junction demands by default.
  - The adjustment of a tank's minimum volume (`Vmin`) when its parameters are changed using `EN_setnodevalue` or `EN_settankdata` has been corrected. 
- - A pump whose status is set to CLOSED in the input file now also has its speed setting set to zero which allows a simple pressure control activate the pump correctly.
- - A failure to raise an error condition for a non-positve pipe roughness in the input file has been fixed.
+ - A pump whose status is set to CLOSED in the input file now also has its speed setting set to zero which allows a simple pressure control to activate the pump correctly.
+ - A failure to raise an error condition for a non-positive pipe roughness in the input file has been fixed.
  - The calculation of head loss gradient for low flow conditions was corrected.
- - Improved updating and convergence tests were added to pressure dependent demand analysis.
+ - Improved updating and convergence tests were added to pressure-dependent demand analysis.
  - Improved checks to prevent outflow from empty tanks or inflow to full (non-overflow) tanks, including the case where a link is connected to a pair of tanks, were added.
  - The CI regression test protocol was modified by:
    - changing the absolute tolerance used to compare the closeness of test results to benchmark values from 0 to 0.0001
    - dropping the "correct decimal digits" test 
    - dropping the check for identical status report content since it prevents accepting code changes that produce more accurate solutions in fewer iterations.
  - A possible loss of network connectivity when evaluating a Pressure Sustaining Valve was prevented.
  - Having the implied loss coefficient for an active Flow Control Valve be less than its fully opened value was prevented.
- - A new type of valve, a Positional Control Valve (PCV), was added that uses a valve characteristic curve to relate its loss coefficient to its fraction open setting. 
- - A new set of functions have been added to get information about upcoming time step events. Users will now see what type of event is going to cause the end of a time step to occur. See `ENtimetonextevent` and `EN_timetonextevent`.
- - A new set of functions have been added to allow users to set a reporting callback function. The user-supplied function will recieve all output normally directed to the report file.
+ - A new type of valve, a Positional Control Valve (PCV), was added that uses a valve characteristic curve to relate its loss coefficient to its fraction open setting (`EN_PCV`).
+ - `EN_VALVE_CURVE` can now be used with the `EN_getcurvetype` and `EN_setcurvetype` to get or set the valve position curve.
```

Comment:
Looking at EN_setcurvetype, I don’t think it’s actually possible to assign EN_VALVE_CURVE because of this line
https://github.com/OpenWaterAnalytics/EPANET/blob/39ffaebcf9458387d2b68dd08b283580dcc43bd4/src/epanet.c#L4910

We might want to open this as a separate issue, but otherwise this line is fine